### PR TITLE
File Credentials Support

### PR DESCRIPTION
**Pull request by m0jt4b4**
- [x]  This commit implements the in-toto metadata siging with Jenkins File Credentials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +44,11 @@ pipeline {
 
   stages {
     stage('Build') {
-      agent { label 'worker 1' }
```
Comment (SantiagoTorres) [inline README.md]: I don't understand the reason for this change? is it that the label doesn't atke space?

File: src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java [hunk 2]
```diff
@@ -99,6 +110,7 @@
      */
     private Link link;
 
+
```
Comment (SantiagoTorres) [inline src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java]: I don't think we need to add this space.

File: README.md (hunk 1 above)
Comment (m0jt4b4) [reply README.md]: No, just wanted it to look closer to the credential Id. Should I change it back?

File: src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java (hunk 2 above)
Comment (m0jt4b4) [reply src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java]: done here : efcc730cabab80004a7d621eb002582a8b567b45

File: README.md (hunk 1 above)
Comment (SantiagoTorres) [reply README.md]: well. I think it keeps the PR more kosher and focused on what this PR really changes.

Comment (m0jt4b4) [reply README.md]: Ok. Changed now 419c572db075a4ead6c1e58c95f059d0db253a95

**Pull request merged**